src/streams/__tests__/streamsReducer-test.js

```diff
@@ -309,22 +309,22 @@ describe('streamsReducer', () => {
       expect(actualState).toEqual(expectedState);
     });
 
-    test('Change the email_address property', () => {
```

Comment:
I'm assuming the only reason this test was passing before was because the update function was using a generic computed property. Now that it's typed, I changed it.

Reply:
Hmmm, and also this test file doesn't have types on it yet. There aren't a lot of files like that left (and all of them are tests -- all our non-test code has had types for years.)

Let's have a prep commit to give it types. See:
https://github.com/zulip/zulip-mobile/blob/main/docs/howto/testing.md#test-style-guide

Reply:
In fact it looks like this is the largest remaining such file!
```
$ git grep -L @flow src/**/*-test.js | xargs wc -l | sort -rn
 2261 total
  368 src/streams/__tests__/streamsReducer-test.js
  359 src/chat/__tests__/flagsReducer-test.js
…
```

So it'll be great to get it converted.